seonshine_frontend/src/pages/login/index.tsx

```diff
@@ -118,7 +146,12 @@ return (
                   alignItems="center"
                 >
                   <FormControlLabel
-                    control={<Checkbox size="small" />}
+                    control={
+                      <Checkbox
+                        size="small"
+                        onChange={rememberMeHandler}
```

Comment:
rename function to `onChangeRememberMe`. I think we should use `useRef` instead of using state